cdap-docs/developers-manual/source/building-blocks/index.rst

```diff
@@ -44,8 +44,8 @@ write data through the data abstraction layer in CDAP.
 - :doc:`MapReduce Programs <mapreduce-programs>`
 - :doc:`Workflows <workflows>`
 - :doc:`Schedules <schedules>`
-- :doc:`Workers <workers>`
```

Comment:
That makes sense, now that we have so many. I have organized the Application "Building Block" portion to be alphabetical.